test/xpu/run_test_with_skip.py

```diff
@@ -922,10 +922,11 @@ def launch_test(test_case, skip_list=None, exe_list=None):
     "test_neg_conj_view_to_sparse_xpu_complex128",
     "test_neg_view_to_sparse_xpu_float64",
 
+    # AssertionError: The supported dtypes for nn.functional.pad on device type xpu are incorrect!
+    # The following dtypes did not work in forward but are listed by the OpInfo: {torch.float16}.
+    # The following dtypes did not work in backward but are listed by the OpInfo: {torch.float16}.
     # Not implemented operator aten::replication_pad2d
```

Comment:
All operators are implemented in XPU, either CPU fallback or XPU implementation?